Sources/NetworkRequestHelper.swift

```diff
@@ -20,13 +20,13 @@ import XCTest
 /// - See also:
 ///    - ``MockNetworkService``
 ///    - ``RealNetworkService``
-class NetworkRequestHelper {
-    private var sentNetworkRequests: [TestableNetworkRequest: [NetworkRequest]] = [:]
+open class NetworkRequestHelper {
+    public var sentNetworkRequests: [TestableNetworkRequest: [NetworkRequest]] = [:]
     /// Matches sent `NetworkRequest`s with their corresponding `HttpConnection` responses.
-    private(set) var networkResponses: [TestableNetworkRequest: [HttpConnection]] = [:]
-    private var expectedNetworkRequests: [TestableNetworkRequest: CountDownLatch] = [:]
+    public private(set) var networkResponses: [TestableNetworkRequest: [HttpConnection]] = [:]
+    public var expectedNetworkRequests: [TestableNetworkRequest: CountDownLatch] = [:]
```

Comment:
same comment here, we should keep these properties private since they are accessible through the APIs below